tests/main/refresh-undo/task.yaml

```diff
@@ -15,17 +15,32 @@ prepare: |
     snapbuild $TESTSLIB/snaps/$SNAP_NAME_GOOD .
     snapbuild $TESTSLIB/snaps/$SNAP_NAME_BAD .
 
+debug: |
+    journalctl -u snap.test-snapd-service.service.service
```

Comment:
But it is the correct name of the snap service (because the service itself is called "service").